tests/foreman/cli/test_errata.py

```diff
@@ -1282,7 +1274,7 @@ def rh_repo_module_manifest(module_entitlement_manifest_org, module_target_sat):
         releasever=None,
     )
     # Sync step because repo is not synced by default
-    rh_repo = entities.Repository(id=rh_repo_id).read()
+    rh_repo = module_target_sat.api.Repository(id=rh_repo_id).read()
```

Comment:
change from `entities.ENTITY` to `target_sat.api.ENTITY` in existing code base is not recommended, you can do it either way but you need to add `*_target_sat` fixture in test module defination like you did it on [line 1149](https://github.com/SatelliteQE/robottelo/pull/13288/files#diff-4536c99de842713bc04ca962d8e94d7c8a5159dca7769ea5cf10b36741ee7cd2R1149)

Reply:
As this is `rh_repo_module_manifest` fixture, would we not want to keep on `module_target_sat`, 
for module scope?

Reply:
> @vijaysawant can you explain what you mean by "change from entities.ENTITY to target_sat.api.ENTITY in existing code base is not recommended"?

I meant to say, if we are working on existing code base then use of `target_sat.api.Repository` on all the places of `entities.Repository` is not mandatory to do, off course we can do it but it will highlight more lines for the review. this change is going to implement in another way by Omkar.
If we are writting any new test case then we should include `*_target_sat` fixture

Reply:
@vijaysawant Right, I do like to change these while I am changing that test/method, as long as everything still tests the same. 
Do you know if Omkar will make use of `*_target_sat.api/cli.Entitity()`, or something different?
A few of the other `entity` calls in this module remain, as some tests and methods did not need fixes, so left them as is.